src/domino/cli/cli.py

```diff
@@ -265,6 +267,34 @@ def generate_random_repo_name():
     return f"new_repository_{str(uuid.uuid4())[0:8]}"
 
 
+@click.command()
+@click.pass_context
+@click.option(
+    '--name',
+    default="ExamplePiece",
+    help='Creates a new piece folder inside the piece repository.'
+)
+def cli_create_piece(ctx, name: str):
+    """Create piece."""
+    try:
+        if not (Path.cwd() / "pieces").is_dir():
+            # might be called inside the pieces directory
+            if Path.cwd().name == "pieces":
+                pieces_repository.create_piece(str(Path.cwd()), name)
+            else:
+                raise FileNotFoundError("No pieces directory found.")
+        else:
+            pieces_repository.create_piece(str(Path.cwd() / "pieces"), name)
+    except FileNotFoundError as err:
+        console.print(err, style=f"bold {COLOR_PALETTE.get('error')}")
+
+@click.group()
+@click.pass_context
+def cli_pieces(ctx):
+    console.print("Manage piece folders.")
```

Comment:
We can add a docstring to be shown in the CLI interface.
Something like:
```suggestion
def cli_pieces(ctx):
    """Manage pieces in a repository."""
    console.print("Manage piece folders.")
```